source/WebApp/components/internal/codemirror/mode-asm.ts

```diff
@@ -2,41 +2,14 @@

```

Comment:
This generator makes surprising choices in a few places, e.g. `(?:d|s)` instead of `[ds]` which would be around 2 times shorter. Or `(?:|x)` instead of `x?` which would be 3 times shorter.

It would be interesting to see if they plan to improve it, alternatively we can run a few post-processing steps.

Reply:
I have opened a PR for the first issue https://github.com/spadgos/regex-combiner/pull/2.
With "regex-combiner": "spadgos/regex-combiner#master" (master is at https://github.com/spadgos/regex-combiner/commit/eb00084) in packages.json, the file size of `src/WebApp/components/internal/codemirror/mode-asm-instructions.ts` is `11419` bytes.
With "regex-combiner": "am11/regex-combiner#feature/char-class", it is `9888` bytes.

We can make similar improvements for other patterns.

Reply:
Updated package.json to point to my fork's branch, for now.